.github/markdown-link-check-config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ignorePatterns": [
+    {
+      "pattern": "^https://www.ftrack.com"
```

Comment:
[question] Can this suppression be more specific to the 520 error? i.e. if a 500 error is returned, then it's not ignored.

Reply:
We can mark specific status codes as "OK" in [the config](https://github.com/tcort/markdown-link-check#config-file-format) but it applies to _all_ URLs. I was tempted to do that, but went with targeting the specific URL instead.

There is an argument that _any_ 5xx code should be "OK" from the perspective of just checking that links are valid. But there are cases where a malformed link, coupled with poor validation on the server side, could trigger a server error code path. So on balance I thought blanket suppressing the specific link, rather than error code, was the lesser evil.

Happy to hear any thoughts on this?

Reply:
I imagine it's whether such as in https://github.com/tcort/markdown-link-check that says a 200 return code is the only way to say a link is alive or not.
But as it's just checking Markdown, I'm not going to make a fuss. (I just had it in my mind pytest tests for `requests` where it's useful to check the return code)